src/bootstrap/CHANGELOG.md

```diff
@@ -1,71 +0,0 @@
-# Changelog
```

Comment:
If you are deleting the changelog, maybe also delete the `changelog-seen` key in config.toml? This key and the changelog was used when we made significant changes to the defaults of bootstrap affecting everyone's workflows.